beacon_node/beacon_chain/src/builder.rs

```diff
@@ -954,13 +954,14 @@ where
     }
 }
 
-impl<TEth1Backend, TEthSpec, THotStore, TColdStore>
-    BeaconChainBuilder<Witness<TestingSlotClock, TEth1Backend, TEthSpec, THotStore, TColdStore>>
+impl<TEth1Backend, TEthSpec, THotStore, TColdStore, TSlotClock>
+    BeaconChainBuilder<Witness<TSlotClock, TEth1Backend, TEthSpec, THotStore, TColdStore>>
 where
     THotStore: ItemStore<TEthSpec> + 'static,
     TColdStore: ItemStore<TEthSpec> + 'static,
     TEth1Backend: Eth1ChainBackend<TEthSpec> + 'static,
     TEthSpec: EthSpec + 'static,
+    TSlotClock: SlotClock + 'static,
```

Comment:
I think we should git rid of this generic because this impl block just adds a `testing_slot_clock` method so doesn't really make sense for it to be usable with the system time slot clock